2022q3/freebsd-foundation.adoc

```diff
@@ -10,98 +10,65 @@ link:https://www.FreeBSDfoundation.org/news-and-events/[Foundation News and Even
+Finally, we provide legal support to the Project when needed and protect the FreeBSD trademarks.
+
+Our goal this year is to raise at a minimum $1,400,000 towards a spending budget of around $2,000,000.
+As we enter the last quarter of 2022, our donation total sits at $167,348, so we still need your help.
+If you haven't made a donation this year, please consider making one at https://freebsdfoundation.org/donate/.
+We also have a Partnership Program for larger commercial donors.
+You can find out more at https://freebsdfoundation.org/our-donors/freebsd-foundation-partnership-program/
 
 ==== OS Improvements
 
-During the third quarter of 2022, 300 src, 36 ports, and 13 doc tree commits
-were made that identified The FreeBSD Foundation as a sponsor.  Some of that
-work has dedicated report entries.
+During the third quarter of 2022, 300 src, 36 ports, and 13 doc tree commits were made that identified The FreeBSD Foundation as a sponsor.
+Some of that work has dedicated report entries.
 
 * FreeBSD as a Tier I cloud-init Platform
 * Intel wireless towards 11ac
 * LLDB multiprocess debugging support
 * OpenStack on FreeBSD
 * Snapshots on Filesystems Using Journaled Soft Updates
 
-The other sponsored work is challenging to concisely summarize.  It varies from
-complex new features to various bug fixes spanning the src tree.  Here is a
-small sample to give a flavor of last quarter's work.
-
-- 240afd8 makefs: Add ZFS support
-
-  This allows one to take a staged directory tree and create a file consisting
-  of a ZFS pool with one or more datasets that contain the contents of the
-  directory tree.  This is useful for creating virtual machine images without
-  using the kernel to create a pool; "zpool create" requires root privileges and
-  currently is not permitted in jails.  makefs -t zfs also provides reproducible
-  images by using a fixed seed for pseudo-random number generation, used for
-  generating GUIDs and hash salts.  makefs -t zfs requires relatively little by
-  way of machine resources.
+The other sponsored work is challenging to concisely summarize.
+It varies from complex new features to various bug fixes spanning the src tree.
+Here is a small sample to give a flavor of last quarter's work.
 
--  36f1526 Add experimental 16k page support on arm64
+- 240afd8 makefs: Add ZFS support +
++
+This allows one to take a staged directory tree and create a file consisting of a ZFS pool with one or more datasets that contain the contents of the directory tree. This is useful for creating virtual machine images without using the kernel to create a pool; `zpool create` requires root privileges and currently is not permitted in jails. `makefs -t zfs` also provides reproducible images by using a fixed seed for pseudo-random number generation, used for generating GUIDs and hash salts. `makefs -t zfs` requires relatively little by way of machine resources.
```

Comment:
I see you are also changing the file to have one sentence per line (by the way, it would be nice to read about it in the commit message: you might want to amend it and then force-push the commit). However, here and in the other paragraphs that you have fixed below I see multiple sentences on the same line: is there a reason for this? Does your fix only work if you keep those sentences on the same line?

Reply:
In a nutshell: hard link breaks, used in this way, are not a proper fix. 

----

> … sentences on the same line?

In this case, with two hard line breaks, AsciiDoc soft line breaks _can_ be added (pictured below): 

![image](https://user-images.githubusercontent.com/192271/213585583-d2226209-16fc-4d6e-956a-eb178a18fd03.png)

In another recent case (I can't recall where) I might have seen only one hard line break. 

In both cases: there's risk of ambiguity. Probably more so where only one hard line break is used. 

----

Beyond the preview in GitHub, beyond an end result at www.freebsd.org, there's also: 

* the plain text email. 

Whilst that's not a consideration for 2022q3 (the email was sent long ago), it probably _should_ be a consideration wherever a plain text email will be part of the workflow. I guess, someone will need to manually wrap and indent wherever an indent has been 'faked' with a hard line break at the tail of a list item. 

----

So … we _can_ add soft line breaks, but (without them) the long unbroken paragraph is like a red flag – to think twice before faking indents in this way.

Reply:
Ok, thanks for clarifying. At least for now, we will do things simple avoiding any potential ambiguity. I am going to merge your pull request.